sway-lsp/src/handlers/notification.rs

```diff
@@ -28,6 +28,7 @@ pub async fn handle_did_open_text_document(
     // Otherwise, don't recompile the project when a new file in the project is opened
     // as the workspace is already compiled.
     if session.token_map().is_empty() {
+        dbg!();
```

Comment:
A few `dbg!`s to remove from this file :smile:

Reply:
Done.